packages/base-types/src/runtimeLogs/logs/kinds.ts

```diff
@@ -34,3 +37,42 @@ export enum StepLogKind {
 export enum GlobalLogKind {
   CONVERSATION_START = 'conversation_start',
 }
+
+const NODE_TYPE_TO_STEP_LOG_KIND = {
```

Comment:
this is an incomplete list and i would greatly appreciate help in expanding it by adding new items to `StepLogKind` and mapping them here. this is used to map our internal `NodeType` values to a human-friendly `StepLogKind` value. ex. `IF` and `IF_V2` correspond to the `CONDITION` step

Reply:
Can always add to this as needed during log creation, too.